sycl/plugins/level_zero/usm_allocator.cpp

```diff
@@ -359,6 +459,44 @@ void Slab::freeChunk(void *Ptr) {
 
 bool Slab::hasAvail() { return NumAllocated != getNumChunks(); }
 
+auto Bucket::getAvailFullSlab() -> decltype(AvailableSlabs.begin()) {
```

Comment:
The following block of code is missing comments, please add

Reply:
OK